cuda/test/matrix/coo_kernels.cpp

```diff
@@ -204,5 +205,18 @@ TEST_F(Coo, AdvancedApplyAddToDenseMatrixIsEquivalentToRef)
     ASSERT_MTX_NEAR(dresult, expected, 1e-14);
 }
 
+TEST_F(Coo, ConvertToDenseIsEquivalentToRef)
+{
+	set_up_apply_data();
+
+	auto dense_mtx = gko::matrix::Dense<>::create(ref);
+	auto ddense_mtx = gko::matrix::Dense<>::create(cuda);
```

Comment:
`const`

Reply:
My bad, it should actually not be const because the result is written in there...